### DISCUSSION TITLE
Default content not respected when switching entry type

### DISCUSSION THREAD

**Discussion by olivierbon** [craftcms/cms#8795] [Ideas]
### Description

If you have a section with multiple entry types (each entry type with different fields), switching entry type from your entry form will not retrieve default field content (probably due to the initial draft).

### Steps to reproduce

1. Create 2 fields (dropdown for example) and set the default value to be the second option in the field (so you can see the problem later)
2. Create a section with two entry types
3. Attach a different dropdown to each entry type
4. Go to Entries → New Entry - On this initial form, the default value for your dropdown is respected
5. Switch entry types - the default for your field is ignored

### Additional info

- Craft version: 3.4.9
- PHP version: 7.3

**Comment by brandonkelly** [comment]
Related: #2446
